README.md

```diff
@@ -105,54 +105,43 @@ Create the following services:
 
 Launch the **WKS** tool and create a new **workspace**.
 
-## 4. Upload Type System
-
-A type system allows us to define things that are specific to review documents. The type system controls how content can be annotated by defining the types of entities that can be labeled and how relationships among different entities can be labeled.
-
-## 5. Import Corpus Documents
-
-Corpus documents are required to train our machine-learning annotator component. For this Code Pattern, the corpus documents will contain example review documents.
-
-## 6. Create an Annotation Set
-
-Once the corpus documents are loaded, we can start the human annotation process. This begins by dividing the corpus into multiple document sets and assigning the document sets to human annotators (for this Code Pattern, we will just be using using one document set and one annotator).
-
-From the **Access & Tools -> Documents** panel, press the **Create Annotation Sets** button. Select a valid **Annotator** user, and provide a unique name for **Set name**.
-
-## 7. Create a Task for Human Annotation
+![](doc/source/images/create-wks-workspace.png)
 
-Add a task for human annotation by creating a task and assigning it annotation sets.
 
-From the **Access & Tools -> Documents** panel, select the **Task** tab and press the **Add Task** button.
+## 4. Upload Type System
 
-### 7.1 Start the Human Annotation task
+A type system allows us to define things that are specific to review documents. The type system controls how content can be annotated by defining the types of entities that can be labeled and how relationships among different entities can be labeled.
 
-Click on the task card to view the task details panel.
+To upload our pre-defined type system, from the Access & Tools -> Entity Types panel, press the Upload button to import the Type System file [data/types-2aa46ad0-31da-11e8-89a9-efc0f3b77492.json](data/types-2aa46ad0-31da-11e8-89a9-efc0f3b77492.json) found in the local repository.
```

Comment:
We usually do:
`Access & Tools` -> `Entity Types`
and `Upload` etc for buttons and menu items.

Reply:
will do on next pass (promise)